modules/ROOT/pages/procedures.adoc

```diff
@@ -1630,6 +1630,10 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/current/p
 | *Mode* 3+| READ
 |===
 
+[NOTE]
+In Cypher 5, the `propertyTypes` column returns the potential Java types for a given property.
+In Cypher 25, it returns the potential Cypher types instead.
+For information about Cypher's property types, see the link:{neo4j-docs-base-uri}/cypher-manual/25/values-and-typesproperty-structural-constructed/#property-types[Cypher Manual -> Property types].
```

Comment:
Is this correct `values-and-typesproperty-structural-constructed`? No slash between types and property?